Scripts/CI/README_Metadata_StyleCheck/README_style_checker.py

```diff
@@ -35,6 +35,7 @@
     'OpenStreetMap',
     'Play a KML Tour',
     'SwiftUI'
```

Comment:
```suggestion
    'SwiftUI',
```
Please add a trailing comma to the previous line.